packages/backend/src/graphql/schema.graphql

```diff
@@ -602,352 +633,402 @@ enum SortOrder {
   assetScale: UInt8!
 }
 
 type OutgoingPaymentConnection {
+  "Pagination information for the outgoing payments."
   pageInfo: PageInfo!
+  "A list of outgoing payment edges, containing outgoing payment nodes and cursors for pagination."
   edges: [OutgoingPaymentEdge!]!
 }
 
 type OutgoingPaymentEdge {
+  "An outgoing payment node in the list."
   node: OutgoingPayment!
+  "A cursor for paginating through the outgoing payments."
   cursor: String!
 }
 
 input OutgoingPaymentFilter {
+  "Filter for outgoing payments based on the receiver's details."
   receiver: FilterString
+  "Filter for outgoing payments based on the wallet address ID."
   walletAddressId: FilterString
+  "Filter for outgoing payments based on their state."
   state: FilterString
 }
 
 type OutgoingPayment implements BasePayment & Model {
-  "Outgoing payment id"
+  "Unique identifier of the outgoing payment."
   id: ID!
-  "Id of the wallet address under which this outgoing payment was created"
+  "Unique identifier of the wallet address under which the outgoing payment was created."
   walletAddressId: ID!
   "Information about the wallet address of the Open Payments client that created the outgoing payment."
   client: String
-  "Available liquidity"
+  "Current amount of liquidity available for this outgoing payment."
   liquidity: UInt64
-  "Outgoing payment state"
+  "State of the outgoing payment."
   state: OutgoingPaymentState!
+  "Any error encountered during the payment process."
   error: String
+  "Number of attempts made to send an outgoing payment."
   stateAttempts: Int!
-  "Amount to send (fixed send)"
+  "Amount to send (fixed send)."
   debitAmount: Amount!
-  "Amount to receive (fixed receive)"
+  "Amount to receive (fixed receive)."
   receiveAmount: Amount!
-  "Wallet address URL of the receiver"
+  "Wallet address URL of the receiver."
   receiver: String!
   "Additional metadata associated with the outgoing payment."
   metadata: JSONObject
-  "Quote for this outgoing payment"
+  "Corresponding quote for the outgoing payment."
   quote: Quote
-  "Amount already sent"
+  "Amount already sent."
   sentAmount: Amount!
-  "Date-time of creation"
+  "The date and time that the outgoing payment was created."
   createdAt: String!
-  "Id of the Grant under which this outgoing payment was created"
+  "Unique identifier of the grant under which the outgoing payment was created."
   grantId: String
 }
 
 enum OutgoingPaymentState {
-  "Will transition to SENDING once payment funds are reserved"
+  "The payment is reserving funds and will transition to `SENDING` once funds are secured."
   FUNDING
-  "Paying, will transition to COMPLETED on success"
+  "The payment is in progress and will transition to `COMPLETED` upon success."
   SENDING
-  "Successful completion"
+  "The payment has been successfully completed."
   COMPLETED
-  "Payment failed"
+  "The payment has failed."
   FAILED
-  "Payment cancelled"
+  "The payment has been canceled."
```

Comment:
American vs british spelling for 1 vs 2 L's?

Reply:
Yeah that's the difference here. I'm ignoring the British spelling of the OutgoingPaymentState enum value of `CANCELLED` 😃